### PR TITLE
Implement missing builtin reductions (#11)

### PR DESCRIPTION
**Pull request by ayberkt**

### CODE REVIEW

**Comment by ayberkt** [comment]
With 0c8d32bb220f9635a00ee9bdd44a2c939bfa5dfa that finalizes the implementation of `intToByteString`, all builtins should now be implemented which fixes #11.

File: src/plutus-core-execution.k [hunk 1]
```diff
@@ -22,20 +23,73 @@ module PLUTUS-CORE-EXECUTION
...
+  rule #hex(10) => "A"
+  rule #hex(11) => "B"
+  rule #hex(12) => "C"
+  rule #hex(13) => "D"
+  rule #hex(14) => "E"
+  rule #hex(15) => "F"
+
+  rule #toHex(0) => ""
+  rule #toHex(N:Int) => #toHex(N /Int 16) +String #hex(N %Int 16)
+    when N =/=Int 0
```
Comment (lucaspena) [inline src/plutus-core-execution.k]: this will fail on `0`

File: src/plutus-core-execution.k [hunk 2]
```diff
@@ -22,20 +23,73 @@ module PLUTUS-CORE-EXECUTION
...
+    when F1 <Float F2
+  rule ( builtin lessThanFloat:Name ( F1:Float F2:Float ) ) => #primFalse
+    when notBool (F1 <Float F2)
+  rule ( builtin equalsFloat:Name   ( F1:Float F2:Float ) ) => #primTrue
+    when F1 ==Float F2
+  rule ( builtin equalsFloat:Name   ( F1:Float F2:Float ) ) => #primFalse
+    when F1 =/=Float F2
+
+  syntax String ::= #toHex(Int)      [function]
+                 | #hex(Int) [function]
```
Comment (lucaspena) [inline src/plutus-core-execution.k]: fix spacing

File: src/plutus-core-syntax.k [hunk 3]
```diff
@@ -13,9 +13,13 @@ module PLUTUS-CORE-SYNTAX
 
   // Module name.
   syntax ModName ::= r"[A-Z][a-zA-Z0-9\\_']*"                    [token, avoid]
+                   /*| #parseToken(String, String)*/
+                                          /*[function, hook(STRING.string2token)]*/
 
   // Constructor name.
   syntax Con     ::= r"[A-Z][a-zA-Z0-9\\_']*"                    [token, avoid]
+                   /*| #parseToken(String, String)
+                                          [function, hook(STRING.string2token)]*/
```
Comment (lucaspena) [inline src/plutus-core-syntax.k]: delete these comments

**Pull request merged**